mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlSource.java

```diff
@@ -58,18 +58,70 @@ protected String createConnectionString() {
     @Nullable
     public Boolean autoReconnect;
 
+    @Name(MysqlConstants.USE_COMPRESSION)
```

Comment:
Do all of the mysql plugins have the same properties with the same descriptions? If so, can another class be added with the common properties?

Reply:
With existing class hierarhy for config classes we have to extend specific parent classes for different plugin types:
1) `DBSpecificQueryActionConfig` and `DBSpecificQueryConfig` for action/post-action plugins.
2) `DBSpecificSinkConfig` for sink plugins.
3) `DBSpecificSourceConfig` for source plugins.

So, there is no way to add a single class with common MySQL properties. We could change existing hierarchy or do not use it for mysql-plugin but it will lead to duplication of common source/sink/action configs between plugins' modules. For example, we would duplicate code from `DBSourceConfig`(https://github.com/data-integrations/database-plugins/blob/develop/database-commons/src/main/java/io/cdap/plugin/db/batch/source/AbstractDBSource.java#L284).

Reply:
Hm I see, makes sense to have it the way you've done it then.